aircraft/breguet_range.py

```diff
@@ -1,8 +1,7 @@
 """Breguet range model"""
-from gpkit.shortcuts import Var, Model
+from gpkit import Variable, Model
 from gpkit.tools import te_exp_minus1
 
-
```

Comment:
for what it's worth, my pep8 checker complains if there are not two lines here. haha.